plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/messages.properties

```diff
@@ -1,13 +1,14 @@
+AppEngineLibraryContainerResolverJobName=Initialize libraries
+BundleContextNotFound=No bundle context was found for service lookup
```

Comment:
This seems like an odd message to present to the user.  We usually don't translate internal error messages since it's unlikely anybody on our team knows Mandarin/Swahili/etc.